app/main.py

```diff
@@ -1 +1,10 @@
-# write your code here
+def copy_file(command: str) -> None:
+    cmd, source_file, target_file = command.split()
```

Comment:
This line could potentially raise a `ValueError` if the `command` string does not contain exactly two spaces. It's a good practice to handle such cases. You could check the length of the resulting list after splitting and return early if it's not 3.